### PR TITLE
nix: init home-manager module

### PR DESCRIPTION
**Pull request by NotAShelf**

### CODE REVIEW

File: nix/hm-module.nix [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+  cfg = config.services.hypridle;
+in {
+  options.services.hypridle = {
+    enable = mkEnableOption "Hypridle, Hyprland's idle daemon";
+
+    package = mkOption {
+      description = "The hypridle package";
+      type = package;
+      default = self.packages.${pkgs.stdenv.system}.hypridle;
```
Comment (fufexan) [inline nix/hm-module.nix:21]:
```suggestion
      default = self.packages.${pkgs.stdenv.hostPlatform.system}.hypridle;
```
Only the latter works when aliases are disabled.

File: nix/hm-module.nix [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+    };
+
+    timeout = {
+      description = "The timeout for the hypridle service";
```
Comment (fufexan) [inline nix/hm-module.nix:25]:
```suggestion
      description = "The timeout for the hypridle service, in seconds";
```

**Review by fufexan — CHANGES_REQUESTED** [review]

**Comment by NotAShelf** [comment]
Also adds the new options that vaxry decidedt o add while we're working.

**Review by fufexan — APPROVED** [review]

**Comment by fufexan** [comment]
CI failing due to missing dep. Merging anyway.

**Pull request merged**